app/controllers/api/v1/study_files_controller.rb

```diff
@@ -361,13 +361,13 @@ def chunk
         content_range = RequestUtils.parse_content_range_header(request.headers)
         if content_range.present?
           if content_range[:first_byte] == 0
-            render json: {errors: 'create/update should be used for uploading the first chunk'}, status: :unprocessable_entity
+            render json: {errors: 'create/update should be used for uploading the first chunk'}, status: :unprocessable_entity and return
           end
           if content_range[:first_byte] != @study_file.upload_file_size
-            render json: {errors: "Incorrect chunk sent: expected bytes starting with #{@study_file.upload_file_size}, received #{content_range[:first_byte]}" }, status: :unprocessable_entity
+            render json: {errors: "Incorrect chunk sent: expected bytes starting with #{@study_file.upload_file_size}, received #{content_range[:first_byte]}" }, status: :unprocessable_entity and return
           end
         else
-          render json: {errors: 'Missing Content-Range header'}, status: :unprocessable_entity
+          render json: {errors: 'Missing Content-Range header'}, status: :unprocessable_entity and return
```

Comment:
Tiny nit - after doing a quick search, it seems that `:bad_request` is the (somewhat) agreed upon response for a missing & required header.

From: https://stackoverflow.com/questions/10576898/what-is-the-most-appropriate-http-status-code-to-return-if-a-required-header-is